.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      NOKOGIRI_USE_SYSTEM_LIBRARIES: true
+      JEKYLL_ENV: development
+      TRAVIS_CI: true
```

Comment:
Can we use a different variable that's more generic? I don't remember what this is doing in the application code but maybe it can be combined with the JEKYLL_ENV environment variable?

Reply:
Ah I see this now in `script/build`. I think setting `JEKYLL_ENV` to `ci` and then updating line 15 accordingly would do the trick.

Reply:
Hey, not sure if I'm following this. Do you mean updating line 15 in the test workflow? Or line 15 in the script/build (both deal with environment stuff).

Reply:
I've pushed up a revision that I think is removing the unnecessary TRAVIS_CI variable, but it does change the build script slightly